src/System.Private.CoreLib/src/System/Diagnostics/StackFrame.CoreRT.cs

```diff
@@ -139,5 +139,7 @@ internal void AppendToStackTrace(StringBuilder builder)
                 builder.AppendLine(DeveloperExperience.Default.CreateStackTraceString(_ipAddress, _needFileInfo));
             }
         }
+
+        private bool IncludeFileInfoIfAvailable() => (_method != null) || AppendStackFrameWithoutMethodBase(sb);
```

Comment:
This won't compile. There is not `sb` local here. Delete this method, keep the code in `ToString()` as what it used to be in CoreRT, and instead add method like this to StackFrame.CoreCLR.cs:

```
private bool AppendStackFrameWithoutMethodBase(StringBuilder sb) => false;
```